addons/revolutionaries/CfgGroups_East.hpp

```diff
@@ -0,0 +1,544 @@
+                rank = "PRIVATE";
+                position[] = {-10, -5, 0};
+            };
+            class unit5 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Base";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {15, -5, 0};
+            };
+        };
+		
+        class TACU_Revolutionaries_G_O_Tanoan_SquadH {
+            name = "Squad (Heavy)";
+            side = 0;
+            faction = "TACU_Revolutionaries_O";
+            icon = "\a3\ui_f\data\map\markers\nato\o_inf.paa";
+            class unit0 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman10";
+                side = 0;
+                rank = "SERGEANT";
+                position[] = {0, 0, 0};
+            };
+            class unit1 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman07";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {5, -5, 0};
+            };
+            class unit2 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman01";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {-5, -5, 0};
+            };
+            class unit3 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman09";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {10, -5, 0};
+            };
+            class unit4 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Base";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {-10, -5, 0};
+            };
+            class unit5 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman10";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {15, -5, 0};
+            };
+        };
+		
+        class TACU_Revolutionaries_G_O_Tanoan_SupportL {
+            name = "Support (Light)";
+            side = 0;
+            faction = "TACU_Revolutionaries_O";
+            icon = "\a3\ui_f\data\map\markers\nato\o_inf.paa";
+            class unit0 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman05";
+                side = 0;
+                rank = "SERGEANT";
+                position[] = {0, 0, 0};
+            };
+            class unit1 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman07";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {5, -5, 0};
+            };
+            class unit2 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman08";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {-5, -5, 0};
+	            };
+            class unit3 {
+                vehicle = "TACU_Revolutionaries_U_O_Tanoan_Rifleman06";
+                side = 0;
+                rank = "PRIVATE";
+                position[] = {10, -5, 0}
+			};
```

Comment:
Using tabs instead of spaces. Formatting is also wrong, the closing bracket is too far in on quite a few of these.
```suggestion
            };
```